model-optimizer/extensions/front/kaldi/apply_counts.py

```diff
@@ -111,10 +116,16 @@ def run_before(self):
                 ]
 
     def find_and_replace_pattern(self, graph: Graph):
-        try:
-            counts = read_counts_file(graph.graph['cmd_params'].counts)
-        except Exception as e:
-            raise Error('Model Optimizer is not able to read counts file {}'.format(graph.graph['cmd_params'].counts) +
-                        refer_to_faq_msg(92)) from e
-
+        # if empty string in counts, read priors from model itself (on loader stage)
+        if graph.graph['cmd_params'].counts == "":
+            counts = graph.graph['priors'].copy()
```

Comment:
what if empty string is specified but `priors` does not exist in graph dictionary. We need to check this case.
Also could you please add a comment about a format `counts` and example

Reply:
Also, should we check a format of counts to prevent crashes?

Reply:
added assert, counts just array - it is checked in assert, so no need in additional comment